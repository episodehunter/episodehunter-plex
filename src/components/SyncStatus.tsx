import * as React from 'react';

export function SyncStatus(props: {error: string}) {
  if (props.error) {
    return <h1 style={{color: '#ff5f5f', fontSize: '16px'}}>{props.error}</h1>;
  } else {
    return <h1>Ready to sync</h1>;
  }
}
