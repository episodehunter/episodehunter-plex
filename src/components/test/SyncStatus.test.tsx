import * as React from 'react';
import * as renderer from 'react-test-renderer';
import { SyncStatus } from '../SyncStatus';

test('that SyncStatus renders', () => {
  const tree = renderer.create(
    <SyncStatus error="" />
  ).toJSON();
  expect(tree).toMatchSnapshot();
});
