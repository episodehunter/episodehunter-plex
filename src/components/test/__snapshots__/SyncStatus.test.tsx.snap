// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`that SyncStatus renders 1`] = `
<h1>
  Ready to sync
</h1>
`;
